### DISCUSSION TITLE
Bootstrap node does not finish, installation does not progress

### DISCUSSION THREAD

**Discussion by mcazeneuve** [okd-project/okd#1727] [General]
**Describe the bug**

On a fresh install  openshift-install-linux-4.13.0-0.okd-2023-09-03-082426
The bootstrap node does not move forward with the installation. It remains waiting for a process to finish.

**Version**

openshift-install-linux-4.13.0-0.okd-2023-09-03-082426
UPI install method, Vsphere 

**How reproducible**

100%

**Log bundle**

journalctl -b -f -u bootkube.service

io.openshift.build.commit.ref=openshift-4.13, io.openshift.expose-services=, io.openshift.build.commit.date=, io.openshift.tags=openshift,base, io.openshift.build.commit.message=, License=GPLv2+)
Sep 06 15:10:18 zn-bts bootkube.sh[655300]: Starting cluster-bootstrap...
Sep 06 15:10:18 zn-bts bootkube.sh[655300]: Waiting for CEO to finish...
Sep 06 15:10:19 zn-bts bootkube.sh[656079]: Error: name "wait-for-ceo" is in use: container already exists
Sep 06 15:10:19 zn-bts podman[656079]: 2023-09-06 15:10:19.021736558 +0000 UTC m=+0.047552096 image pull 03ab13fdb63234058e1f4d89423a6022e722a14202179bdbc7c11bb0b57f34d9 quay.io/openshift/okd-content@sha256:ac0516ca2287bd7535f252665354db21866a265053e4ae3e62d9a1d79c2f6411
Sep 06 15:10:19 zn-bts systemd[1]: bootkube.service: Main process exited, code=exited, status=125/n/a
Sep 06 15:10:19 zn-bts systemd[1]: bootkube.service: Failed with result 'exit-code'.
Sep 06 15:10:19 zn-bts systemd[1]: bootkube.service: Consumed 5.816s CPU time.

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Comment by mcazeneuve** [comment]
[bootstrap-collect-log.txt](https://github.com/okd-project/okd/files/12540438/bootstrap-collect-log.txt)
[openshift_install.log](https://github.com/okd-project/okd/files/12540442/openshift_install.log)

**Reply by mcazeneuve** [reply]
Hi vrutkovs , please tell me if you need anything more to analyze this case.

**Reply by vrutkovs** [reply]
Does `openshift-install gather bootstrap` work? This would collect more information

**Reply by mcazeneuve** [reply]
it does not work
 openshift-install gather bootstrap --dir instalacion --log-level debug --bootstrap 192.168.1.200
DEBUG OpenShift Installer 4.13.0-0.okd-2023-09-03-082426 
DEBUG Built from commit efdf6fb258d835614fef0d570074defa5d74aa93 
DEBUG Fetching Bootstrap SSH Key Pair...           
DEBUG Loading Bootstrap SSH Key Pair...            
DEBUG Using Bootstrap SSH Key Pair loaded from state file 
DEBUG Reusing previously-fetched Bootstrap SSH Key Pair 
INFO Skipping VM console logs gather: no platform configured in metadata 
INFO Pulling debug logs from the bootstrap machine 
DEBUG Using SSH_AUTH_SOCK /tmp/ssh-XXXXaN4v87/agent.43276 to connect to an existing agent 
FATAL failed to create SSH client: failed to use pre-existing agent, make sure the appropriate keys exist in the agent for authentication: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
